src/xdocs/config_misc.xml

```diff
@@ -923,7 +923,7 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
           </tr>
           <tr>
             <td>ignorePrimitiveTypes</td>
-            <td>ignore primitive types as parameters</td>
+            <td>ignore primitive types as parameters.</td>
```

Comment:
Haven't we been making these descriptions start with an uppercase letter.

Reply:
done